### PR TITLE
Add xplat sample for customscript

### PR DESCRIPTION
**Pull request by bingosummer**
Refine README
Update Azure CLI sample

### CODE REVIEW

File: CustomScript/README.md [hunk 1]
```diff
@@ -16,57 +15,95 @@ the download directory of the scripts, and the tail of the output is
...
-## Requirement
-Python 2.7+
-## Usage
-PowerShell Sample
+## Version
+1.3
+
+## Configuration
+### Public Configuration
+```javascript
```
Comment (thomas1206) [inline CustomScript/README.md]: javascript?

Comment (bingosummer) [inline CustomScript/README.md]: Yes, for json format.

File: CustomScript/README.md [hunk 2]
```diff
@@ -74,4 +111,4 @@ $TimeStamp = (Get-Date).Ticks
 ```
 
 ## More Information
-http://azure.microsoft.com/blog/2014/08/20/automate-linux-vm-customization-tasks-using-customscript-extension/
\ No newline at end of file
+To learn more information about the usage of CustomScrpt Extension, please visit [Automate Linux VM Customization Tasks Using CustomScript Extension](http://azure.microsoft.com/blog/2014/08/20/automate-linux-vm-customization-tasks-using-customscript-extension/).
```
Comment (thomas1206) [inline CustomScript/README.md]: one typo: CustomScrpt -> CustomScript

File: CustomScript/README.md (hunk 1 above)
Comment (thomas1206) [inline CustomScript/README.md]: OK.

**Comment by bingosummer** [comment]
Will open a new pull request after adding more document for ARM.

**Pull request closed without merging**